ДЗ.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Седона</title>
+</head>
+
+<body>
+
+<hedar>
+ <nav>
+  <a>
+      <img src="img/welcome.png" width="456.65" height="350.88" alt="Логотип «Седона»">
```

Comment:
Нужно указывать целые числа без дробей, иначе невалидно
```suggestion
      <img src="img/welcome.png" width="456" height="350" alt="Логотип «Седона»">
```